samples/msmq/persistence/sample.md

```diff
@@ -0,0 +1,24 @@
+---
+title: MSMQ Subscription Persistence usage
+reviewed:
+component: MsmqPersistence
+related:
+- persistence/msmq
+---
+
+
+## Prerequisites
+
+Ensure that MSMQ has been installed.
+
+
+## Code walk-through
+
+This sample shows basic usage of MSMQ as storage for subscriptions. The application publishes an empty message to itself, via the MSMQ transport, and writes to the console when the message is received. 
+
+Note that the default queue name, `NServiceBus.Subscriptions`, is overriden to avoid sharing the subscription storage queue with other endpoints on the same machine. The queue name is `{EndpointName}.Subscriptions`.
+
+
+### Configuration
```

Comment:
@andreasohlund - I think it might be important to add a note that, you have to use other persistence options for Timeouts as timeouts is not supported via MSMQ like for subscription storage. InMemory while ok for the sample is not a good production safe option and a note will clarify that much better. Thoughts?

Reply:
Done, just added a generic note about storage needs. The docs for InMemory already have the relevant "don't use in production" warnings (https://docs.particular.net/persistence/in-memory)